frontend/src/pages/simplecss.css

```diff
@@ -47,4 +47,9 @@ a .list-group-item-text {
     border-radius: 4px;
     margin-bottom: 20px;
     cursor: pointer;
+}
+.resetButton {
+    background-color: #f9cbc2 !important;
```

Comment:
Usually `!important` isn't good practice. We should probably use one of [these methods](https://material-ui.com/customization/components/) for styling a reset button.